bldsys/cmake/sample_helper.cmake

```diff
@@ -253,4 +266,38 @@ endif()
             set_property(TARGET ${bare_name}-${extension} PROPERTY FOLDER "HLSL_Shaders")
         endforeach()
     endif()
+
+    # Slang shader compilation
+    if(Vulkan_slang_EXECUTABLE AND DEFINED SHADERS_SLANG)
+        foreach(SHADER_FILE_SLANG ${TARGET_SHADERS_SLANG})
+            get_filename_component(SLANG_SPV_FILE ${SHADER_FILE_SLANG} NAME_WLE)
+            get_filename_component(bare_name ${SLANG_SPV_FILE} NAME_WLE)
+            get_filename_component(extension ${SLANG_SPV_FILE} LAST_EXT)
+            get_filename_component(directory ${SHADER_FILE_SLANG} DIRECTORY)
+            string(REGEX REPLACE "[.]+" "" extension ${extension})
+            set(OUTPUT_DIR "${CMAKE_CURRENT_BINARY_DIR}/shader-slang-spv")
+            set(OUTPUT_FILE ${OUTPUT_DIR}/${bare_name}.${extension}.spv)
+            file(MAKE_DIRECTORY ${OUTPUT_DIR})
+
+            set(SLANG_PROFILE "glsl_460")
```

Comment:
Why limit to glsl?  Can just emit directly with spirv_1_4

Reply:
Thx, will take a look at that. I took that from glslangs official documentation ;)